client/js/components/auth/login.js

```diff
@@ -12,44 +12,72 @@ const login = (state, prev, send) => {
 
   var prefix = css`
```

Comment:
Ah yeah, so declaring this inline is causing the duplicate CSS issue. Each time this function is called, a new line of CSS is appended to head, which causes the performance problems.

When using `bankai` we use the `extract-css` transform which takes this declaration and turns it into a static string which means it's no problem. If you move this outside the scope e.g. to line 4 it should all be good (: